domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/FrequencyStartTimeAfterEndTimeNotice.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright (c) 2020. MobilityData IO.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.notice.error;
+
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.NoticeExporter;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.ErrorNotice;
+
+import java.io.IOException;
+
+public class FrequencyStartTimeAfterEndTimeNotice extends ErrorNotice {
+    public FrequencyStartTimeAfterEndTimeNotice(final String filename,
+                                                final String startTimeAsString,
+                                                final String endTimeAsString,
+                                                final String tripId) {
+        super(filename,
+                E_046,
+                "Fields `start_time` and `end_time` out of order",
+                "`end_time`: `" + startTimeAsString + "` precedes `start_time`: `" +
+                        endTimeAsString + "` in file `" + filename +
```

Comment:
I'd suggest always putting the file name last in the error log. This is because most developers and those familiar with GTFS will know the file just by looking at the field names, so the key info that they need (fields that are wrong and the ID fields) should go first as they are more important and needed for every error.

Reply:
Will do so.